recipes/epics-alh/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+export TOP="${EPICS_BASE}/extensions"
+export MOTIF_INC="${PREFIX}/include"
+export MOTIF_LIB="${PREFIX}/lib"
+export X11_INC="${PREFIX}/include"
+export X11_LIB="${PREFIX}/lib"
+
+_make="make -j1 V=1 VERBOSE=1 EPICS_BASE=${EPICS_BASE} TOP=${TOP}"
+
+# build
+${_make} build
+
```

Comment:
Is there a make check or test we can run here?

Reply:
@ocefpaf @duncanmmacleod I don't believe so. I think the problem here may be the same as the one for epics-striptool and other epics extensions: https://github.com/conda-forge/staged-recipes/pull/18283#discussion_r825749741